templates/base.html

```diff
@@ -102,8 +102,7 @@
             <a class="nav-link" style="color: white" href="{% url 'home' %}"><i class="fas fa-home"></i> Home</a>
           </li>
           <li class="nav-item active">
-            <a class="nav-link" style="color: white" href="#">            
-            <i class="fas fa-info-circle"></i> About</a>
+            <a class="nav-link" style="color: white" href="{% url 'about' %}"><i class="fas fa-info-circle"></i> About</a>
```

Comment:
This links to the about page section for a non-logged in user, we should also add this code to the nav bar for a logged-in user @induramesh